Copy Jepsen tests for Cassandra and Scalar DB (#1)
scalar-labs/scalar-jepsen: README.md

# Run tests with Jepsen Docker
1. Clone [Jepsen repository](https://github.com/jepsen-io/jepsen)
2. Configure to mount `scalar-jepsen` directory by editing `${JEPSEN}/docker/docker-compose.dev.yml` like the following

```yaml
   control:
     volumes:

>        - ${JEPSEN_ROOT}:/jepsen # Mounts $JEPSEN_ROOT on host to /jepsen control container

Author: `${JEPSEN_ROOT}` is defined by another file.
I changed the name into the consistent name `${JEPSEN_ROOT}`.

>        - ${SCALAR_JEPSEN}:/scalar-jepsen

Reviewer: `${SCALAR_JEPSEN_ROOT}` for consistency ?

```

3. Start docker with `--dev` option

```sh
$ cd ${JEPSEN}/docker
$ ./up.sh --dev
```

4. Run tests in `jepsen-control`
  - You can login `jepsen-control` by the following command
  ```sh
  $ docker exec -it jepsen-control bash
  ```

```
# cd /scalar-jepsen/cassandra
# lein run test --test lwt
```

  - Check README in each test for more detail

# Run tests without Docker
1. Launch debian machines as a control machine and Cassandra nodes

>   - We recommend 1 control machine and 5 node's machines

Reviewer 2: ```suggestion
  - We recommend 1 control machine and 5 node machines
```

Author: Fixed

>     - You can decrease the number of nodes. If it's decreased, you need to specify the nodes when a test starts.

Reviewer 2: ```suggestion
    - You can decrease the number of nodes. If you do so then you will need to specify the nodes when starting a test.
```

Author: Fixed

2. Install Java8 on each machine
```sh
sudo apt install openjdk-8-jre
```

3. Install Leiningen (https://leiningen.org/) on the control machine
4. Make a SSH key pair on the control machine

> 5. Add the public key to login nodes as root on each node

Author: It's for Jepsen to login nodes. I modified the sentence.
